ODLA/platforms/CMakeLists.txt

```diff
@@ -57,6 +57,11 @@ if (ODLA_BUILD_VODLA)
   add_subdirectory(vodla)
 endif()
 
+option(ODLA_BUILD_MAGICMIND "Build ODLA Runtime Library for Magicmind" ON)
```

Comment:
let's set the default to "OFF" now because SDK is not installed on the CI server.